src/middelware/ce_to_legacy_event.ts

```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 import {Request, Response, NextFunction} from 'express';
-import {isBinaryCloudEvent, getBinaryCloudEventContext} from '../cloudevents';
+import * as cloudevents from '../cloudevents';
```

Comment:
I tend to dislike star imports. Could we keep named imports?

I think with the above `CE_SERVICE` change, the number of imports will be manageable.